gravitee-apim-console-webui/src/management/api/list/api-list.component.spec.ts

```diff
@@ -94,22 +95,40 @@ describe('ApisListComponent', () => {
         tags: 'Tags',
       },
     ]);
-    expect(rowCells).toEqual([['', '🪐 Planets', 'play_circlecloud_done', '/planets', '', 'admin']]);
+    expect(rowCells).toEqual([['', '🪐 Planets', 'play_circlesync_problemcloud_done', '/planets', '', 'admin']]);
   }));
 
   it('should order rows by name', fakeAsync(async () => {
-    const apis = [fakeApi({ name: 'Planets 🪐' }), fakeApi({ name: 'Unicorns 🦄' })];
+    const planetsApi = fakeApi({ id: '1', name: 'Planets 🪐' });
+    const unicornsApi = fakeApi({ id: '2', name: 'Unicorns 🦄' });
+    const apis = [planetsApi, unicornsApi];
     await initComponent(apis);
 
     const nameSort = await loader.getHarness(MatSortHeaderHarness.with({ selector: '#name' })).then((sortHarness) => sortHarness.host());
     await nameSort.click();
+    apis.map((api) => expectSyncedApi(api.id, true));
     expectApisListRequest(apis, null, 'name');
 
     fixture.detectChanges();
     await nameSort.click();
+    apis.map((api) => expectSyncedApi(api.id, true));
     expectApisListRequest(apis, null, '-name');
   }));
 
+  it('should display out of sync api icon', fakeAsync(async () => {
+    const api = fakeApi();
+    const apis = [api];
+    await initComponent(apis);
+
+    const nameSort = await loader.getHarness(MatSortHeaderHarness.with({ selector: '#name' })).then((sortHarness) => sortHarness.host());
+    await nameSort.click();
+
+    expectSyncedApi(api.id, false);
+    expect(await loader.getHarness(MatIconHarness.with({ selector: '.states__api-is-not-synced' }))).toBeTruthy();
```

Comment:
WDYT to Check this synced icon is not displayer before expectSyncedApi ?

Reply:
Done